ci/gdk_build.template.steps.yaml

```diff
@@ -13,7 +13,7 @@ common: &common
     - "platform=windows"
     - "permission_set=builder"
     - "scaler_version=2"
-    - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v4-20-03-03-102720-bk8971-dd78adbb}"
+    - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v4-20-03-11-114147-bk9341-57494374-d}"
```

Comment:
Same here with this queue: https://github.com/spatialos/UnrealGDK/pull/1942 Might go in first - if so we drop this change.